BraveWallet/Panels/Encryption/EncryptionView.swift

```diff
@@ -80,9 +80,16 @@ struct EncryptionView: View {
         VStack(spacing: 8) {
           Blockie(address: request.address)
             .frame(width: min(blockieSize, maxBlockieSize), height: min(blockieSize, maxBlockieSize))
-          Text(account.name)
-            .font(.subheadline.weight(.semibold))
-            .foregroundColor(Color(.secondaryBraveLabel))
+          AddressView(address: account.address) {
+            VStack(spacing: 4) {
+              Text(account.name)
+                .font(.subheadline.weight(.semibold))
+                .foregroundColor(Color(.braveLabel))
+              Text(account.address.truncatedAddress)
+                .font(.subheadline.weight(.semibold))
+                .foregroundColor(Color(.secondaryBraveLabel))
+            }
+          }
```

Comment:
do we want the account name to be included for the context menu?

Reply:
I think it's okay; it gives a bigger area for long press to see the context menu and lets the user see their account name with the associated address. Does it create issues/confusion with the copy menu? It can be just the address if warranted.

Reply:
i have no preference. either way is fine. its just some other places only the address has the context menu.